src/lib/typeHelpers.ts

```diff
@@ -106,3 +106,13 @@ export function getIsStringUnion<T extends string>(
     });
   };
 }
+
+export function isObjectWithStringMessage(
+  unknown: unknown
+): unknown is { message: string } {
```

Comment:
Why not `unknown is Error`?

Reply:
Because it's not true. I am not checking if it's an error here. That would be done with instanceof. I am thinking maybe you are right. I should just use instanceof. I am just inventing unnecessary generaliztions here